pinot-common/src/test/java/org/apache/pinot/common/utils/config/TableConfigSerDeTest.java

```diff
@@ -251,7 +249,8 @@ public void testSerDe()
     {
       // with upsert config
       UpsertConfig upsertConfig =
-          new UpsertConfig(UpsertConfig.Mode.FULL, null, "comparison", UpsertConfig.HashFunction.NONE);
+          new UpsertConfig(UpsertConfig.Mode.FULL, null, UpsertConfig.Strategy.OVERWRITE, "comparison",
```

Comment:
For `FULL` upsert, pass in `null` as the default strategy as that does not apply? Same for other places